veloroutes_voies_vertes/install/sql/veloroutes/70_COMMENT.sql

```diff
@@ -504,6 +504,10 @@ COMMENT ON COLUMN veloroutes.statut_segment_val.code IS 'Code';
 COMMENT ON COLUMN veloroutes.statut_segment_val.libelle IS 'Libellé';
 
 
+-- VIEW v_portion
+COMMENT ON VIEW veloroutes.v_portion IS 'vue qui joint les portions aux collections de géométries des segments qui les composent';
```

Comment:
```suggestion
COMMENT ON VIEW veloroutes.v_portion IS 'Vue qui joint les portions aux collections de géométries des segments qui les composent';
```

Juste parce que les autres commentaires sont aussi avec une majuscule. Un commentaire sur les autres vues?